.github/workflows/e2e_test.yml

```
@@ -0,0 +1,25 @@
+      - name: Checkout repository
+        uses: actions/checkout@v4
+
+      - name: Setup environment
+        run: |
+          sudo apt-get update
+          sudo apt-get install -y gcc make
+
+      - name: Run Bash Script
+        run: |
+          chmod +x ./test/e2e/runner.sh
+          ./test/e2e/runner.sh
```

yamllint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)